public/app/features/dashboard/containers/NewDashboardPage.tsx

```diff
@@ -25,6 +26,13 @@ export default function NewDashboardPage(props: GrafanaRouteComponentProps) {
   return showDashboardPage ? (
     <DashboardPage {...props} />
   ) : (
-    <DatasourceOnboarding onNewDashboard={() => setCreateDashboard(true)} loading={loading} />
+    <DatasourceOnboarding
```

Comment:
Super small naming suggestion. WDYT?
```suggestion
    <EmptyStateNoDatasource
```